pkg/controllers/iaa/controller.go

```diff
@@ -188,8 +188,7 @@ func (c *controller) NewDaemonSet(rawObj client.Object) *apps.DaemonSet {
 	yes := true
 	daemonSet := apps.DaemonSet{
 		ObjectMeta: metav1.ObjectMeta{
-			//Namespace:    devicePlugin.Namespace,
-			Namespace:    metav1.NamespaceSystem,
+			Namespace:    c.ns,
```

Comment:
this whole function needs be updated to what other controllers have:
https://github.com/intel/intel-device-plugins-for-kubernetes/blob/959cef517f620ff0ecc523e250d89056fe916733/pkg/controllers/dsa/controller.go#L174-L181